broker-core/src/main/java/io/zeebe/broker/transport/GatewayComponent.java

```diff
@@ -27,11 +29,14 @@
 
   @Override
   public void init(final SystemContext context) {
-    final SocketBindingGatewayCfg gatewayCfg =
-        context.getBrokerConfiguration().getNetwork().getGateway();
+    final NetworkCfg network = context.getBrokerConfiguration().getNetwork();
+    final SocketBindingGatewayCfg gatewayCfg = network.getGateway();
     if (gatewayCfg.isEnabled()) {
       try {
+        final SocketBindingClientApiCfg clientApiCfg = network.getClient();
         final Gateway gateway = new Gateway(gatewayCfg.getHost(), gatewayCfg.getPort());
+        // TODO(menski): help this is horrible
```

Comment:
Do you need help? :sweat_smile:

Reply:
I think we will address this in the gateway configuration issue #1135